hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -187,15 +190,19 @@ protected Statement withAfters(FrameworkMethod method, Object target, Statement
         List<FrameworkMethod> afters = getTestClass().getAnnotatedMethods(After.class);
         Statement nextStatement = statement;
         List<TestRule> testRules = getTestRules(target);
+        ExpectedException expectedException = null;
         if (!testRules.isEmpty()) {
             for (TestRule rule : testRules) {
                 if (rule instanceof BounceMemberRule) {
                     nextStatement = ((BounceMemberRule) rule).stopBouncing(statement);
                 }
+                if (rule instanceof ExpectedException) {
+                    expectsException = true;
```

Comment:
Should this be expectedException = rule ?

Reply:
good catch, this line was left in a separate changeset in my IDE